### PR TITLE
Jordan/1674 created PageProfile for shared stylez

### LINKED ISSUE #1674: wrong class on PageProposal elements
Description:

`PageProposal` is using `validator-profile...` classes, for eg `class="validator-profile__header validator-profile__section proposal"` .

We should change them to have `proposal-page...` classes

### PR DESCRIPTION
**Pull request by jbibla**
Closes #1674 

_Description:_

- created PageProfile for shared stylez
- fixed problematic naming of validator classes on proposal elements

❤️ Thank you!

---

- [x] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [x] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

File: app/src/renderer/components/staking/PageValidator.vue [hunk 1]
```diff
@@ -70,16 +70,14 @@
...
               <dt>Voting Power</dt>
-              <dd id="validator-profile__power">
-                {{ pretty(powerRatio * 100) }} %
-              </dd>
+              <dd id="page-profile__power">{{ pretty(powerRatio * 100) }} %</dd>
             </dl>
             <dl v-if="config.devMode" class="colored_dl">
               <dt>Uptime</dt>
-              <dd id="validator-profile__uptime">
+              <dd id="page-profile__uptime">
```
Comment (faboweb) [inline app/src/renderer/components/staking/PageValidator.vue]: are you sure about name spacing these under `page-profile`?

Comment (jbibla) [reply app/src/renderer/components/staking/PageValidator.vue]: i think it's an ok name, intended to describe our pages with a "profile-like" layout. open to suggestions if you have them.

Comment (sabau) [reply app/src/renderer/components/staking/PageValidator.vue]: If we recycle all this I'm happy! 
The `uptime` case seems a bit contextualized in validators although, everything else seems general enough to be recycled in other pages.

Ideally I would keep separate (declare in validator page) only those three classes:
- self-bond 
- commissions
- uptime

Comment (jbibla) [reply app/src/renderer/components/staking/PageValidator.vue]: yeah, i agree with you @sabau - i didn't actually write or refactor any css in this PR - just renamed things. i would be happy to improve the css here to make it more generalized or i could do this in another PR ... whatever you guys think is best.

Comment (faboweb) [reply app/src/renderer/components/staking/PageValidator.vue]: I am fine with doing it in another PR. Going to merge this. Can you open a follow up?

**Pull request merged**